EIPS/eip-1962.md

```diff
@@ -0,0 +1,190 @@
+---
+eip: 1962
+title: EC arithmetics and pairings with runtime definitions
+author: Alex Vlasov (@shamatar)
+discussions-to: https://github.com/ethereum/EIPs/pull/1962
+type: Standards Track
+category: Core
+status: Draft
+created: 2019-04-22
+requires: 1109
+---
+
+# Simple summary
+
+This proposal is an extension and formalization of EIP1829 with an inclusion of pairings. EIP1109 is required due to possibly very low cost of point addition operation compared to the STATICCALL opcode.
```

Comment:
Is EIP1109 strictly required or will it just reduce the gas cost of these op codes?

Reply:
I’d say that it’s strictly required. Point addition would have a cost of 100 gas for a curve with not too long base field, while calling a precompile would cost 500 gas for a STATICCALL opcode

Reply:
That being said, the real comparison is how much that would cost on EVM directly, if it is more than `700 + 100` then the precompile is still cheaper and likely that is the case.

I'd say this requirement on 1109 (or reduced cost to calling precompiles, in general) is not a strict requirement, but something which increases usability a lot.